src/content/docs/all-the-things.md

```diff
@@ -0,0 +1,92 @@
+---
+title: All the things
+---
+
+FreeCodeCamp has created numerous projects over the years and this is a list of
+everything we've done so far. A link to every project has been provided and a list
+of the technology stack the project is using.
+
+## Active Projects
+
+### [Our Learning Platform](https://www.freecodecamp.org)
+
+The site that started it all. Originally launched back in 2014; our website is sitll going
+strong to this day. It offers lessons and curriculum in order to provide everyone with a free
+online platform to learn how to write code and engage with the industry. The source code can be found on [GitHub](https://github.com/freeCodeCamp/freeCodeCamp)
+
+Our website is powered by the following technology stack:
+
+- [NodeJS](https://nodejs.org/en) : the Javascript runtime for our platform and a majority of
+  our projects
+
+- [MongoDB](https://www.mongodb.com/) : this NoSQL database storage solution is responsible
+  for storing and maintaining all of our camper's records
+
+- [GatsbyJS](https://www.gatsbyjs.com/) : this static website solution lets us serve the cirriculum pages
+  to the campers
+
+- [NextJS](https://nextjs.org/) : this React powered framework is responsible for handling our API.
+
+### [Our Web 3.0 Curriculum](https://web3.freecodecamp.org/)
+
+Thanks to sponsorships, freeCodeCamp offers specialized content that discusses blockchain and cryptocurrency. The source code can be found on [GitHub](https://github.com/freeCodeCamp/web3-site)
+
+### [Our forums](https://forum.freecodecamp.org/)
+
+Our forums where campers can discuss troubles with the challenges and various other topics related to development. The Discourse platform hosts all of the content and makes this experience possible.
```

Comment:
Related repo is the forum-theme one https://github.com/freeCodeCamp/forum-theme

Reply:
Thank you. I was unaware it had a repository.